test/core/kernel_gateway/prove_block_opening.js

```diff
@@ -38,53 +39,24 @@ contract('KernelGateway.proveBlockOpening()', async (accounts) => {
     updatedWeights,
     auxiliaryBlockHash,
     storageBranchRlp,
-    accountRlp,
-    accountBranchRlp,
     originBlockHeight,
     kernelHash,
     transitionHash;
 
-  async function setAccountProof(isValid) {
-    accountRlp = testData.account.rlpEncodedAccount;
```

Comment:
`testData` is not more used. Remove from `require`.

Reply:
Eagle eyes 👀

Reply:
😳